services/worker/src/worker/job_runners/split/opt_in_out_urls_scan_from_streaming.py

```diff
@@ -125,26 +128,25 @@ async def check_spawning(
 
 async def opt_in_out_task(
     image_urls: List[str], session: ClientSession, semaphore: Semaphore, limiter: AsyncLimiter, spawning_url: str
-) -> Tuple[List[Any], List[Any]]:
+) -> Any:
     try:
         spawning_response = await check_spawning(image_urls, session, semaphore, limiter, spawning_url)
     except Exception:
         raise ExternalServerError(message=f"Error when trying to connect to {spawning_url}")
     if "urls" not in spawning_response:
         raise ExternalServerError(message=f"Error when trying to connect to {spawning_url}: '{spawning_response}'")
-    opt_in_urls_indices = [i for i in range(len(image_urls)) if spawning_response["urls"][i]["optIn"]]
-    opt_out_urls_indices = [i for i in range(len(image_urls)) if spawning_response["urls"][i]["optOut"]]
-    return opt_in_urls_indices, opt_out_urls_indices
+    return spawning_response
 
 
 async def opt_in_out_scan_urls(
     urls: List[str],
+    urls_columns: List[str],
     urls_number_per_batch: int,
     spawning_token: str,
     max_concurrent_requests_number: int,
     max_requests_per_second: int,
     spawning_url: str,
-) -> Tuple[List[int], List[int]]:
+) -> Tuple[List[OptUrl], int, int]:
```

Comment:
Personal taste (feel free to ignore): I prefer using named tuples, TypedDict, or dataclasses (in the same way I prefer keyword arguments over positional arguments in the call to functions) to be more explicit